tsdb/engine/tsm1/writer.go

```diff
@@ -652,6 +668,19 @@ func (t *tsmWriter) Close() error {
 	return nil
 }
 
+// Remove removes any temporary storage used by the writer.
+func (t *tsmWriter) Remove() error {
+	if err := t.index.Remove(); err != nil {
+		return err
+	}
+
+	if f, ok := t.wrapped.(*os.File); ok {
+		f.Close()
```

Comment:
In this case we're going to ignore a `Close` error (fine) but then still try and remove the file. This seems at odds with how the `directIndex` is handled.

Reply:
I'll fix this up too.